ra_aid/agent_utils.py

```
@@ -545,6 +569,14 @@ def run_planning_agent(
     if thread_id is None:
         thread_id = str(uuid.uuid4())
 
+    # Get latest project info
+    try:
+        project_info = get_project_info(".")
+        formatted_project_info = format_project_info(project_info)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)